rls-quick-start-guide.md

```diff
@@ -0,0 +1,59 @@
+# RLS Quick-Start Guide
+
+Hello, RLS members new and old! Welcome to the Rands Leadership Slack quick-start guide 🙂 
+
+RLS is a big community! The last time Rands made an #announcement, RLS membership was the same size as East Peoria, Illinois with 22,620 people. As RLS continues to grow, it can be harder to figure out where to get started. This quick-start guide is intended to give you that starting point to exploring and participating in this community. 
+
+And please be assured that we do want you - _yes, you!_ - to participate. You are not too junior, or too new, or not leader-y enough to contribute. Our community benefits from being able to discuss and learn from all of our perspectives and experiences, and it's what we love about being in RLS.
+
+## Code of Conduct
+
+Please make sure you read and understand the RLS [Code of Conduct](https://github.com/randsleadershipslack/documents-and-resources/blob/main/code-of-conduct.md). We respect it and take it quite seriously!
+
+## Welcome / Tackling Imposter Syndrome
+
+Remember what we said a minute ago about wanting you to participate here? We really do mean it - but we also understand that can be easier said than done. 
+
+This is a big place, and it can feel intimidating. On the surface, Rands Leadership Slack looks like a group of high-performing leaders with extensive experience and a wide range of expertise. And that's true - but that's not all we are. We treat leadership as a human quality, not just a job title. We make mistakes and learn from each other. We support and encourage each other. We're a community. 
```

Comment:
From this para onward, how about this:

> This is a big place, and it can feel intimidating. On the surface, Rands Leadership Slack looks like a group of high-performing leaders with extensive experience and a wide range of expertise. And that's true - but that's not all we are. We are humans with strengths and weaknesses. Our community is a fabric of many individuals from different backgrounds, companies, countries and working cultures. We each bring different perspectives to the discussion - and so can you.
>
>We know what imposter syndrome is. Many of us experience it regularly in some form. You may well feel it too upon joining. Finding yourself amongst a self-selecting group of leaders, often in high profile roles, you can suddenly feel smaller than you did before. We strongly encourage you to push past this. Start contributing. Find your own voice and share your own insights. Ask questions where you want to learn. We welcome you and hope you will quickly find engaging with us rewarding.
>
>We treat leadership as a human quality, not just a job title. We make mistakes and learn from each other. We support and encourage each other. We're a community.

Reply:
❤️